.github/workflows/apply-version-label-issue.yml

```diff
@@ -0,0 +1,15 @@
+name: Apply version label to issue
+
+on:
+  issues:
+    types: [opened, edited]
+
+jobs:
+  add-version-label-issue:
+    runs-on: ubuntu-latest
+    continue-on-error: true
```

Comment:
There is supposed to be a ["Version: Unspecified" label](https://github.com/facebook/react-native/labels/Version%3A%20unspecified) it uses in this case -- but I forgot why this wasn't working -- @cortinico do you have an issue in which the script was failing? Or the error again? It might've been an internal task?

Reply:
Sadly I can't find it now on top of my head as the previous action was removed. Essentially the GH actions was returning a failure instead of having a fallback which should be  "Version: Unspecified" as you suggested 👍

Reply:
I tested this by creating an issue without specifying the version https://github.com/gabrieldonadel/react-native/issues/7 on my fork and as you can see here everything succeeded https://github.com/gabrieldonadel/react-native/runs/4929422660?check_suite_focus=true, are there any other scenarios that I should test?

Reply:
LGTM 👍